app/page.client.tsx

```diff
@@ -1,13 +1,17 @@
 'use client';
 
-import { Heading } from '@chakra-ui/react';
+import { Center, Flex, Heading } from '@chakra-ui/react';
+import ModuleLaunchPad from './ModuleLaunchPad';
 
 const PageClient = () => (
-  <>
-    <Heading as="h2" size="lg">
-      Welcome to EDPN!
-    </Heading>
-  </>
+  <Center width="100%">
+    <Flex flexDirection="column" gap="24px" width="100%" maxWidth="1500px">
```

Comment:
Not critical for this PR but just sharing for future reference. 

Would Flex `justify-content` with `center` not work? Infact you can just put `<Flex flexDirection="column" gap="24px">` and remove the Center component and other properties, I pulled your code and tried it, the component doesn't seem to do anything that Flex isn't already handling. Don't worry about positioning or limiting the width, we can worry about those once we have specific layouts planned. Keep prop use limited in general until needed.